README.md

```diff
@@ -111,12 +111,10 @@ before starting work on new features.
     $ make serve
     ~~~
 
-[bower]: http://bower.io/
 [contact-address]: mailto:team@carpentries.org
 [django]: https://www.djangoproject.com
 [fixing-npm-permissions]: https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npm-s-default-directory-to-another-directory
 [issues]: https://github.com/swcarpentry/amy/issues
-[nodejs]: https://nodejs.org/
-[npm]: https://www.npmjs.com/
```

Comment:
I'm not a fan of mixing various changes under one commit.

You should put changes to the Yarn instruction in a separate commit (not under "Add support for Python3.4+"), or squash it back into "Add instructions to install yarn, not bower".

This is minor change, don't bother with it now, just remember in future.